src/test/java/hw1/DivisionTest.java

```diff
@@ -0,0 +1,75 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DivisionTest {
+
+    private Calculator calc = new Calculator();
+
+    //    @BeforeTest
+    //    public void before_class(){
+    //         calc = new Calculator();
+    //    }
+
+    @Test(groups = {"multDivGroup"}, dataProvider = "DataForDivision",
+            dataProviderClass = DataProviders.class)
+    public void divideLongTest(Object[] l) {
```

Comment:
In test methods you can use signature `multiplyLongTest(long a, long b, long expectedRes)` which is more readable and TestNG will try automatically cast data from DataProvider to arguments

Using `o`, `l` for variable names is not recommended, because these names sometimes can be misread like zero and one

Reply:
Thank you for this advice. It really improves readability  as well as reduces the overall number of code lines